src/modules/logicModule/LM_PC_MigrateLiquidity_UniswapV2_v1.sol

```diff
@@ -0,0 +1,230 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity 0.8.23;
+
+// Internal Interfaces
+import {IOrchestrator_v1} from
+    "src/orchestrator/interfaces/IOrchestrator_v1.sol";
+import {ILM_PC_MigrateLiquidity_UniswapV2_v1} from
+    "@lm/interfaces/ILM_PC_MigrateLiquidity_UniswapV2_v1.sol";
+import {IERC20PaymentClientBase_v1} from
+    "@lm/abstracts/ERC20PaymentClientBase_v1.sol";
+
+// Internal Dependencies
+import {
+    ERC20PaymentClientBase_v1,
+    Module_v1
+} from "@lm/abstracts/ERC20PaymentClientBase_v1.sol";
+import {IFM_BC_Bancor_Redeeming_VirtualSupply_v1} from
+    "@fm/bondingCurve/interfaces/IFM_BC_Bancor_Redeeming_VirtualSupply_v1.sol";
+import {IBondingCurveBase_v1} from
+    "@fm/bondingCurve/interfaces/IBondingCurveBase_v1.sol";
+
+// External Interfaces
+import {IUniswapV2Router02} from "@ex/interfaces/uniswap/IUniswapV2Router02.sol";
+import {IUniswapV2Factory} from "@ex/interfaces/uniswap/IUniswapV2Factory.sol";
+import {IERC20} from "@oz/token/ERC20/IERC20.sol";
+import {IERC20Issuance_v1} from "src/external/token/IERC20Issuance_v1.sol";
+
+// External Dependencies
+import {ERC165Upgradeable} from
+    "@oz-up/utils/introspection/ERC165Upgradeable.sol";
+
+interface BondingCurve is IBondingCurveBase_v1 {
+    /// @notice Transfer a specified amount of Tokens to a designated receiver address.
+    /// @dev    This function MUST be restricted to be called only by the {Orchestrator_v1}.
+    /// @dev    This function CAN update internal user balances to account for the new token balance.
+    /// @param  to The address that will receive the tokens.
+    /// @param  amount The amount of tokens to be transfered.
+    function transferOrchestratorToken(address to, uint amount) external;
```

Comment:
Instead of the bondingCurve interface you could just use the IFundingManager Interface here.
Contains the same function